Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3204,7 +3204,8 @@ void WebPageProxy::handleMouseEvent(const NativeWebMouseEvent& event)
 void WebPageProxy::dispatchMouseDidMoveOverElementAsynchronously(const NativeWebMouseEvent& event)
 {
     sendWithAsyncReply(Messages::WebPage::PerformHitTestForMouseEvent { event }, [this, protectedThis = Ref { *this }](WebHitTestResultData&& hitTestResult, OptionSet<WebEventModifier> modifiers, UserData&& userData) {
-        this->mouseDidMoveOverElement(WTFMove(hitTestResult), modifiers, WTFMove(userData));
+        if (!this->isClosed())
```

Comment:
Do we really need `this->`?

Reply:
No we don't! Removed `this->` for better readability in https://github.com/WebKit/WebKit/commit/2b9bcceb47a78b4416bc5cfd9e869284d25e0e16